maintainers/maintainer-list.nix

```diff
@@ -2900,6 +2900,11 @@
     github = "martingms";
     name = "Martin Gammelsæter";
   };
+  marius851000 = {
```

Comment:
You should rebase on master as another PR of yours has already been merged :tada:, so this is no longer required.